extensions/pdh/tests/PerformanceDataCounterTests.cpp

```diff
@@ -122,8 +122,8 @@ TEST_CASE("PDHCounterDataCollectionTest", "[pdhcounterdatacollectiontest]") {
   REQUIRE(int_counter.collectData());
 
   rapidjson::Document document(rapidjson::kObjectType);
-  double_counter.addToJson(document, document.GetAllocator());
-  int_counter.addToJson(document, document.GetAllocator());
+  double_counter.addToJson(document, document.GetAllocator(), utils::optional<int8_t>());
+  int_counter.addToJson(document, document.GetAllocator(), utils::optional<int8_t>());
```

Comment:
You can use `utils::nullopt` for empty optionals.

Reply:
yeah that makes the code more readable, changed it